### ISSUE TITLE
eslint error - jsx a11y

### ISSUE THREAD

**Issue by jihun1233** [jihun1233/dark-mode-react#4]
eslint 오류가 계속 나는데 해결방안을 찾지는 못함. airbnb설정에서 오류가 있는듯 하지만 정확히 알 수 없음.
> Failed to load plugin 'jsx-a11y' declared in '.eslintrc » eslint-config-airbnb ...

**Comment by jihun1233** [comment]
추가-a11y 설정에서 control-has-associated-label에러 출력
- control element 사용 시 항상 label을 항상 함께 사용해주어야 한다는 것 같은데 이를 off해주는게 좋을듯하다. 매번 label에 nesting하거나 htmlFor을 사용할 순 있지만 label을 강제로 사용한다는 점에서 별로인것같다.
-관련 이슈:https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/566

**Comment by jihun1233** [comment]
추가-control-has-associated-label이 button 태그에서만 문제가 발생하는 이유
- button 태그 내부에 아무 정보없이 그대로 태그를 닫아두었는데 안에 들어가는 텍스트가 label로 취급되는 것 같다. 안에 텍스트 문구를 넣으니 에러가 뜨지 않는다.
- 관련 질문: https://stackoverflow.com/questions/57616720/react-jsx-a11y-eslint-throws-unexpected-errors-for-control-label
- 이전까지는 control-label을 꼭 묶어 사용해야한다는 느낌의 오류였는데 이런 경우에는 button에 아무런 text를 주지 않고 empty button으로 사용할 수 있는지에 대한 옵션인 것 같다.
